addons/hr/hr.py

```diff
@@ -268,12 +268,20 @@ def _broadcast_welcome(self, cr, uid, employee_id, context=None):
                 ('groups_id', 'in', group_id)
             ], context=context)
         partner_ids = list(set(u.partner_id.id for u in res_users.browse(cr, SUPERUSER_ID, user_ids, context=context)))
-        self.message_post(
-            cr, uid, [employee_id],
-            body=_('Welcome to %s! Please help him/her take the first steps with Odoo!') % (employee.name),
-            partner_ids=partner_ids,
-            subtype='mail.mt_comment', context=context
-        )
+        model_obj = self.pool.get('ir.model.data')
+        template_obj = self.pool.get('mail.template')
+        get_template = model_obj.xmlid_to_res_id(cr, uid, 'hr.welcome_hr_new_employee')
+        template_id = template_obj.browse(cr, uid, get_template)
```

Comment:
no need to use model_obj. you can directly use self.pool['ir.model.data'].xmlid_to_res_id()
and use self.pool instead of self.pool.get()